packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts

```diff
@@ -100,18 +91,30 @@ export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
   }
 
   public getSourcePathStrategy(): SourcePathStrategy {
+    // @ts-ignore
```

Comment:
why the `ts-ignore` here?

Reply:
since metadataType is not assigned in the constructor anymore.

Reply:
`this.metadataType!.pathStrategy` could work around this. 
But the real solution is, we should use a getter for metadata type (so that we don't have to repeat the try/catch blocks) & catch exception (programmer error most likely) if metadataType is undefined.

Reply:
I'll use the "!". Not sure if adding a getter is going to provide much help since metadataType will be undefined only due to a programmer error and tests should right away catch it as it did for forceLightningLwcTestCreate.ts when I made the change.

Reply:
yeah, the only thing is if it's missed by the test it will get an error like cannot get suffix of undefined. If we don't expect this class to be used a lot since we are moving to libraries, it's fine using `!`.